docs/routing.md

```diff
@@ -99,13 +97,13 @@ export default {
 
 ### Module's configuration
 
-Make sure you set the `parsePages` option to `false` to disable acorn parsing and add your custom paths in the `pages` option:
+Make sure you set the `parsePages` option to `false` to disable babel parsing and add your custom paths in the `pages` option:
 
 ```js
 // nuxt.config.js
 
 ['nuxt-i18n', {
-  parsePages: false,   // Disable acorn parsing
+  parsePages: false,   // Disable babel parsing
```

Comment:
Just a question here: should we plan on dropping this option at some point? I don't feel too good about having 2 different ways for configuring page-specific stuff with `nuxt-i18n` (in-component options and in Nuxt's config). IMO, having the ability to change `nuxt-i18n`'s behaviour right in the Vue files feels much simpler and I would rather drop the other method if we can guarantee there won't be any compatibility issues like we had with `acorn` parsing. WDYT?

Reply:
I agree that defining those in the vue file itself is much more fitting than doing that in module's options. That said, people hate breaking changes so I would feel a bit bad about dropping the other method... And it's fairly trivial to support module option alternative. So not sure really...

Reply:
Actually, my main concern is the documentation: it's a bit cumbersome to have to maintain documentation for these 2 different approaches, and I think that it makes onboarding harder for people who start using **nuxt-i18n**. I think we're fine for now but we might want to discuss this further for some next major release.

Reply:
Right, then the best way to do it is to deprecate `pages` option (add console warning when used) and remove in next major release.